### PR TITLE
email processing 

### PR DESCRIPTION
**Pull request by sigfrid**
Extract application from GK-Importer.

### CODE REVIEW

File: config/application.rb [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  def self.root
+    File.expand_path('..', __dir__) + '/'
+  end
+
+  def self.env
+    ENV['APP_ENVIRONMENT'] || 'development'
+  end
+end
+
+Bundler.require(:default, App.env)
```
Comment (irismolinari) [inline config/application.rb]: I think there could be a issue here (the same we encountered with pnmas-gateway). Where will APP_ENVIRONMENT be defined? If in the .env file, ENV['APP_ENVIRONMENT'] is loaded after this step and the bundler will run always with 'development'.

**Comment by sigfrid** [comment]
As stated in [README](https://github.com/acsinfo/v570-gateway/blob/setup/README.md#environment-variables) APP_ENVIRONMENT is defined [here](https://github.com/acsinfo/v570-gateway/blob/setup/Dockerfile#L9).

**Comment by irismolinari** [comment]
I've reviewed the pull request, it can be merged but it is opened on master. Shall I to close this one and open a new one on development?

**Comment by sigfrid** [comment]
For this time the branch may be merged into `master` since currently we don't have this app in production. I will update anyway the default branch so future PRs will be created with `development` as base.

**Pull request merged**